roles/rust-foundation-project-director.md

```diff
@@ -0,0 +1,34 @@
+# Role Description: Rust Foundation Project Director
+
+Please refer to [Rust Foundation Director Roles & Responsibilities](TODO) for components of the role description that apply to all Foundation Board Directors.
```

Comment:
Is the linked document here supposed to be the same as the one referenced below, https://foundation.rust-lang.org/static/board-director-role-description.pdf? If not, has this document been written yet?

Reply:
I believe it is. @yaahc can confirm?